src/Features/Core/Portable/UseCollectionInitializer/AbstractUseCollectionInitializerCodeFixProvider.cs

```diff
@@ -102,7 +102,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 subEditor.ReplaceNode(statement, newStatement);
                 foreach (var match in matches)
                 {
-                    subEditor.RemoveNode(match);
+                    subEditor.RemoveNode(match, SyntaxRemoveOptions.KeepUnbalancedDirectives);
```

Comment:
Default options pass in "KeepUnbalanced | UseElasticTrivia".  The formatter removes blank lines when there is elastic trivia.